.github/actions/get-changelog/.eslintignore

```diff
@@ -0,0 +1,4 @@
+dist/
+lib/
+node_modules/
+jest.config.js
```

Comment:
[q] If this ignores basically all files, why is eslint (and prettier) even included in this package?

Reply:
it applies to the ts code of the action. since we don't have a lot of code we can discuss if we need eslint/prettier here, this is the config from github's template and I think all exclusions make sense.

Reply:
I didn't realize that `dist/` and `lib/` does not include the code. My assumption was `lib` is code and `dist` is compile output, but dist was just renamed lib... my bad!